source/utils/ResourceManager.cpp

```diff
@@ -133,6 +134,10 @@ void ResourceManager::setupDataPath(boost::program_options::variables_map& optio
         {
             mGameDataPath.append("/");
         }
+
+#if OGRE_VERSION > 0x10900
```

Comment:
For the reference, this check was incorrect, as it would be true for 1.9.1 and later until 1.10.0.
Fixed in #1303.